js/src/redux/providers/signerMiddleware.js

```diff
@@ -120,7 +120,7 @@ export default class SignerMiddleware {
         });
     }
 
-    handlePromise(this._api.signer.confirmRequest(id, { gas, gasPrice }, password));
+    handlePromise(this._api.signer.confirmRequest(id, { gas, gasPrice }, password.password));
```

Comment:
`views/Signer/components/TransactionPendingForm/TransactionPendingFormConfirm/transactionPendingFormConfirm.js` is passing an Object the the `onConfirm` prop, and `views/Signer/components/SignRequest/signRequest.js` is expecting a string.